packages/zoe/src/contractFacet/zcfMint.js

```diff
@@ -150,21 +143,24 @@ export const makeZCFMintFactory = async (
     ZCFMintFactoryI,
     {
       async makeZCFMintInternal(keyword, zoeMint) {
+        const issuerRecord = await E(zoeMint).getIssuerRecord();
+
         const zcfMintBaggage = makeScalarBigMapStore('zcfMintBaggage', {
           durable: true,
         });
-        const zcfMint = await makeDurableZcfMint(
-          keyword,
-          zoeMint,
-          zcfMintBaggage,
-        );
-        zcfMintBaggageSet.add(zcfMint);
-        return zcfMint;
+
+        zcfMintBaggage.init('issuerRecord', issuerRecord);
```

Comment:
This works, but it now looks awfully like per-`zcfMint` state, and I wouldn't mind a cleanup pass that replaced the `prepareSingleton` (one Kind per mint) with a `prepare` something else that shared a single Kind among all mints, with `state.keyword`, `state.zoeMint`, and `state.issuerRecord`. That would make startup faster, remove the extra Kinds, reduce memory use a little bit, and be generally simpler.

But, probably out of scope for this PR.

Reply:
Good idea. I'll follow up with that.